src/app/components/Staking/Form/StakingAmount.tsx

```diff
@@ -51,17 +53,10 @@ export const StakingAmount: React.FC<StakingAmountProps> = ({
     }
   };
 
-  const handleBlur = (_e: FocusEvent<HTMLInputElement>) => {
-    if (!btcWalletBalanceSat) return;
-    setTouched(true);
-
-    if (value === "") {
-      onStakingAmountSatChange(0);
-      setError(generalErrorMessage);
-      return;
-    }
-
+  useEffect(() => {
+    if (btcWalletBalanceSat === undefined) return;
     const numValue = parseFloat(value);
+    if (!numValue) return;
```

Comment:
previously we were changing the staking amount and setting an error in this condition. Why not now?